src/SIM/SIM2D.C

```diff
@@ -279,6 +279,11 @@ bool SIM2D::parseGeometryTag (const TiXmlElement* elem)
       IFEM::cout <<"\tCollapsed edge P"<< patch <<" E"<< edge << std::endl;
       pch->collapseEdge(edge);
     }
+
+#ifdef USE_OPENMP
+      // Cannot do multi-threaded assembly with collapsed edges
+      omp_set_num_threads(1);
```

Comment:
The indentation here looks wrong. Anyway, a better way of fixing this would be to alter the stripe direction of be aligned with the collapse edge (thought we did that last year when doing the hemisphere problem, but can't find it)